charts/cluster-scanner/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Sysdig Cluster Scanner
 
 type: application
 
-version: 0.6.0
+version: 0.6.1
```

Comment:
Since is a new feature I suggest to bump the minor release instead of the patch release here.

Reply:
Sure, will do, thanks!